miner/worker.go

```diff
@@ -1336,10 +1337,23 @@ LOOP:
 	// when in-turn, compare with remote work.
 	from := bestWork.coinbase
 	if w.bidFetcher != nil && bestWork.header.Difficulty.Cmp(diffInTurn) == 0 {
+		waitBidTimer := time.NewTimer(w.config.DelayLeftOver / 5)
+	LOOP_WAIT_BID:
+		for {
+			select {
+			case <-waitBidTimer.C:
+				break LOOP_WAIT_BID
+			default:
+				if w.bidFetcher.GetSimulatingBid(bestWork.header.ParentHash) == nil {
```

Comment:
This is not good, the CPU is busy with doing `w.bidFetcher.GetSimulatingBid(bestWork.header.ParentHash)`, this may starve other thread.

Reply:
fixed